src/pages/validator/amplifier/verifier-onboarding.mdx

```diff
@@ -1,197 +1,147 @@
+1. Run `tofnd`. 
 
-## Onboard as a verifier
+If you do not have it, you can pull and run the latest with the following:
 
-### Run `tofnd`
+```bash
+docker pull haiyizxx/tofnd:latest
+docker run -p 50051:50051 --env MNEMONIC_CMD=auto --env NOPASSWORD=true -v tofnd:/.tofnd haiyizxx/tofnd:latest
+```
 
-- If you are running `tofnd` for the first time, a verifier key will be generated for you automatically.
+1. Then, pull and run [`ampd`](https://github.com/axelarnetwork/axelar-amplifier/tree/main/ampd#worker-onboarding), the off-chain daemon responsible for voting and signing with the amplifier protocol, needs access to a running `tofnd` instance in order to onboard as a worker or run the daemon. See the [tofnd repository](https://github.com/axelarnetwork/tofnd) for more info.
 
-    ```docker
+1. Download `ampd` through either the [binary](https://github.com/axelarnetwork/axelar-amplifier/releases/tag/ampd-v0.1.0) or [docker](https://hub.docker.com/r/axelarnet/axelar-ampd).
 
-    docker run  -p 50051:50051 --env MNEMONIC_CMD=auto  --env NOPASSWORD=true -v tofnd:/.tofnd haiyizxx/tofnd:latest
-    ```
+For docker, be sure to pull the `0.1.0` version and `amd64` platform specifically, i.e. `docker pull axelarnet/axelar-ampd:v0.1.0 --platform amd64`.
 
-    If you have done this before, don't run this, cmmand, instead restart your existing container by container id.
+1. Create a `config.toml` file and put it in your `~/.ampd/` directory. 
 
-- Now check to ensure it's running successfully
+Example config.toml file for the current devnet:
 
-  ```bash
-  docker ps -a --filter ancestor=haiyizxx/tofnd:latest
-  ```
+```bash
+    tm_jsonrpc="http://devnet-amplifier.axelar.dev:26657"
+    tm_grpc="tcp://devnet-amplifier.axelar.dev:9090"
+    event_buffer_cap=1000
+
+    [service_registry]
+    cosmwasm_contract="axelar1hrpna9v7vs3stzyd4z3xf00676kf78zpe2u5ksvljswn2vnjp3ystlgl4x"
+
+    [broadcast]
+    batch_gas_limit="1000000"
+    broadcast_interval="1s"
+    chain_id="devnet-amplifier"
+    gas_adjustment="2"
+    gas_price="0.00005uamplifier"
+    queue_cap="1000"
+    tx_fetch_interval="600ms"
+    tx_fetch_max_retries="10"
+
+    [tofnd_config]
+    batch_gas_limit="10000000"
+    key_uid="axelar"
+    party_uid="ampd"
+    url="http://127.0.0.1:50051"
+
+    [[handlers]]
+    cosmwasm_contract="axelar1ufs3tlq4umljk0qfe8k5ya0x6hpavn897u2cnf9k0en9jr7qarqqa9263g"
+    type="MultisigSigner"
+
+    [[handlers]]
+    chain_name="avalanche"
+    chain_rpc_url="https://red-divine-dream.avalanche-testnet.quiknode.pro/fe8a71d7e75a6fce0f2232361b795745ae818f6a/ext/bc/C/rpc"
+    cosmwasm_contract="axelar1466nf3zuxpya8q9emxukd7vftaf6h4psr0a07srl5zw74zh84yjq4687qd"
+    type="EvmMsgVerifier"
```

Comment:
missing config for other chains that are available on this devnet, fantom, sepolia, optimism sepolia. would be good to add those as well by default